app/src/main/kotlin/jp/co/yumemi/android/code_check/RepositorySearchViewModel.kt

```
@@ -36,18 +47,25 @@ class RepositorySearchViewModel(application: Application) : AndroidViewModel(app
-            // レスポンスをJSONとしてパース
-            val jsonBody = JSONObject(response.readText())
-            val jsonItems = jsonBody.optJSONArray("items") ?: return@withContext emptyList()
+                // レスポンスをJSONとしてパース
+                val jsonBody = JSONObject(response.readText())
+                val jsonItems = jsonBody.optJSONArray("items") ?: return@withContext emptyList()
 
-            // JSON配列をリストに変換
-            parseRepositoryItems(jsonItems)
+                // JSON配列をリストに変換
+                parseRepositoryItems(jsonItems)
+            } catch (e: JSONException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)